### DISCUSSION TITLE
Complex/conditional view filters?

### DISCUSSION THREAD

**Discussion by dotansimha** [community/community#7726] [Projects and Issues]
We are looking for a way to do a filter like the following for a view:

```
(repo:"a","b") OR (label:"someLabel")
```

or even:

```
(repo:"a","b","c") OR (label:"someLabel",no)
```

Is it possible to get a similar functionality? it seems like the operator between filters today is always `AND` ?

**Comment by jasononeil** [comment]
I mentioned this on a [different issue](https://github.com/github/feedback/discussions/12418) but I'd love to be able to use:

```
assignee:@me OR reviewer:@me
```
